src/VichUploaderBundle.php

```diff
@@ -23,9 +23,4 @@ public function build(ContainerBuilder $container): void
         $container->addCompilerPass(new RegisterFlysystemRegistryPass());
         $container->addCompilerPass(new RegisterSluggerPass());
     }
-
```

Comment:
If this is the only way to solve the issue, I have to reject this PR.
Feel free to close it or to propose an alternate approach